packages/front-end/pages/features/[fid].tsx

```diff
@@ -203,25 +205,79 @@ export default function FeaturePage() {
             >
               Show implementation
             </a>
-            {permissions.createFeatures && (
-              <DeleteButton
-                useIcon={false}
-                displayName="Feature"
+        {permissions.createFeatures && (<DeleteButton
+              useIcon={false}
+              displayName="Feature"
+              onClick={async () => {
+                await apiCall(`/feature/${data.feature.id}`, {
+                  method: "DELETE",
+                });
+                router.push("/features");
+              }}
+              className="dropdown-item"
+              text="Delete feature"
+            />) }
+            {(isArchived && permissions.createFeatures) ? (
+              <ConfirmButton
                 onClick={async () => {
-                  await apiCall(`/feature/${data.feature.id}`, {
-                    method: "DELETE",
+                  await apiCall(`/feature/${data.feature.id}/archive`, {
+                    method: "PUT",
                   });
                   router.push("/features");
                 }}
-                className="dropdown-item"
-                text="Delete feature"
-              />
+                modalHeader="Unarchive Feature"
+                confirmationText={
+                  <>
+                    <p>
+                      Are you sure you want to continue? This will make the
+                      current feature active again.
+                    </p>
+                  </>
+                }
+                cta="Unarchive"
+                ctaColor="danger"
+              >
+                <button className="dropdown-item">Unarchive Feature</button>
+              </ConfirmButton>
+            ) : (
+              <ConfirmButton
+                onClick={async () => {
+                  await apiCall(`/feature/${data.feature.id}/archive`, {
+                    method: "PUT",
+                  });
+                  router.push("/features");
+                }}
+                modalHeader="Archive Feature"
+                confirmationText={
+                  <>
+                    <p>
+                      Are you sure you want to continue? This will make the
+                      current feature unactive.
+                    </p>
+                  </>
+                }
+                cta="Archive"
+                ctaColor="danger"
+              >
+                <button className="dropdown-item">Archive Feature</button>
+              </ConfirmButton>
             )}
           </MoreMenu>
         </div>
       </div>
 
-      <h1>{fid}</h1>
+      <div className="row align-items-center mb-0">
```

Comment:
If you use the `row` class, the children should all be wrapped in divs with a column class (e.g. `<div className="col-auto">...</div>`).  Otherwise, the padding and margins will be messed up.

An alternative is to use `d-flex` instead of `row`, which makes it use flexbox and that's it.

Reply:
Changed in recent commit